lib/generators/administrate/install/install_generator.rb

```diff
@@ -31,6 +37,13 @@ def run_dashboard_generators
         end
       end
 
+      def model_check
+        if valid_dashboard_models.none?
+          puts "** ERROR: Add models before installing Administrate. **"
+          puts "**        Consider removing 'app/controllers/admin'. **"
```

Comment:
Why this message?

Reply:
because, trying to avoid the other methods by an error didn't seem possible. and that is the file that the install generator creates if no valid models are present

Reply:
I feel that it can confuse more than help. I think a reader would think that there's something special in that file that needs to be refreshed every time they run the generator. Additionally, if they are running the generator now, they will later when the have models, so I think it's safe to assume that they'll want the file.

Reply:
maybe, but there is the potential that they would like to use the admin namespace for something specific to their rails app, and now we have potentially forced them to use a different name for it (unless they are comfortable deleting the file). 

I don't know, I feel it is useful to make people comfortable resetting their app to a state before call administrate install. But I trust your opinion more on how the message reads (gets interpreted)

Reply:
I see what you mean. The problem I see is lack of consistency. If you run any other generator and there's an error, any files created so far will stay, and it will be on the developer what to do. Typically you'd use Git, which would point out the changes and provide means for rolling back.

As for the actual message, "consider removing this file", to me is an encouragement strong enough that can send the wrong message, and lead to believe that there's something special to the file. In my case, I think that I would understand that the file is different in every run, perhaps based on the models at the time of running, and perhaps even that it should be re-generated if we add new models.

I'd prefer to remove it.

Reply:
That makes sense, I'll make those changes now